template/tcg-storage.tex

```diff
@@ -0,0 +1,50 @@
+%%
+% Copyright (c) 2017 - 2021, Pascal Wagler;
+% Copyright (c) 2014 - 2021, John MacFarlane
+%
+% All rights reserved.
```

Comment:
Should be covered by the repo LICENSE file so we don't have to add it for this file.

Reply:
I removed Copyright